packages/codecs/oer/src/index.ts

```diff
@@ -2,4 +2,6 @@ import Reader from './lib/reader'
 import Writer, { WriterInterface } from './lib/writer'
 import Predictor from './lib/predictor'
 
+export { default as BaseError } from './errors/base-error'
```

Comment:
This is an example of needing to access the private/impls stuff from a module ...
could potentially make a `private` or `testing` namespace and import from that ...

Reply:
How important is it to keep these private?